public/javascripts/previewers/three_js/viewer_three.js

```diff
@@ -14,6 +14,48 @@
 
 ****************************************************************/
 
+/****************************************************************
+  Wish list:
+
+  1) Become able to upload zip files into clowder which will be 
```

Comment:
there is a zip-extractor that does this.

Reply:
Oops. Sorry, I didn't mean for these changes to be merged. I only wanted you to merge the new `viewer_hop.js`.

Reply:
The other previewer is a work in progress.

Reply:
So, if its possible please merge the `viewer_hop.js` only. Otherwise, I'll redo the process from scratch.